CHANGELOG.latest.md

```diff
@@ -1,28 +1,19 @@
 ## RevenueCat SDK
-### Customer Center
-#### 🐞 Bugfixes
-* Dismiss promotional offer sheet after successful purchase (#4475) via Will Taylor (@fire-at-will)
+
 ### Win-back Offers
-#### ✨ New Features
-* Support Redeeming Win-Back Offers with Streamlined Purchasing Disabled (#4370) via Will Taylor (@fire-at-will)
 
-## RevenueCatUI SDK
-### Customer Center
 #### ✨ New Features
-* Add support for `product_mapping` in promotional offers (#4489) via Cesar de la Vega (@vegaro)
-#### 🐞 Bugfixes
-* Close feedback survey after picking an option (#4444) via Cesar de la Vega (@vegaro)
+
+- Support fetching & redeeming eligible win-back offers in custom paywalls (#4485) via Will Taylor (@fire-at-will)
+
+### 🐞 Bugfixes
+
+- Fix transaction metadata in purchase tester app (#4505) via Will Taylor (@fire-at-will)
```

Comment:
Not important this time, but I feel like changes to Purchase Tester should always be in the "Other" category since they're not really user facing.

Reply:
Definitely!

Reply:
Will do that in future releases :)